_schemaV2.graphql

```diff
@@ -14819,6 +14838,63 @@ enum PartnerOfferSorts {
   END_AT_DESC
 }
 
+type PartnerOfferToCollector implements Node {
+  artworkId: String
+  createdAt(
+    format: String
+
+    # A tz database time zone, otherwise falls back to "X-TIMEZONE" header. See
+    # http://www.iana.org/time-zones,
+    # https://en.wikipedia.org/wiki/List_of_tz_database_time_zones
+    timezone: String
+  ): String
+  endAt(
+    format: String
+
+    # A tz database time zone, otherwise falls back to "X-TIMEZONE" header. See
+    # http://www.iana.org/time-zones,
+    # https://en.wikipedia.org/wiki/List_of_tz_database_time_zones
+    timezone: String
+  ): String
+
+  # A globally unique ID.
+  id: ID!
+
+  # A type-specific ID.
+  internalID: ID!
+  isActive: Boolean
```

Comment:
question (non-blocking): I'm curious if we have a plan in the future to use the `isActive` field anywhere since [the endpoint](https://github.com/artsy/gravity/pull/17374/files) to return partner offers on me only gives us active partner offers.

Reply:
Removed `isActive` since we don't need it right now :+1: